alibi_detect/cd/base_online.py

```diff
@@ -98,7 +99,7 @@ def reset(self) -> None:
         "Resets the detector but does not reconfigure thresholds."
         self._initialise()
 
-    def predict(self, x_t: np.ndarray,  return_test_stat: bool = True,
+    def predict(self, x_t: Union[np.ndarray, Any],  return_test_stat: bool = True,
```

Comment:
Does it make sense to simplify the type hint here (and in other equivalent places) just to 'Any' now?